livekit-rtc/livekit/rtc/_ffi_client.py

```diff
@@ -204,8 +211,13 @@ def instance(cls) -> "FfiClient":
     def __init__(self) -> None:
         self._lock = threading.RLock()
         self._queue = FfiQueue[proto_ffi.FfiEvent]()
-
-        ffi_lib.livekit_ffi_initialize(ffi_event_callback, True)
+        
+        ffi_lib.livekit_ffi_initialize(
+            ffi_event_callback,
+            True,
+            b"python",
+            version("livekit").encode('utf-8')
```

Comment:
let's use our version.py as the ground truth 
from ..version import __version__

Reply:
ah - thank you! didn't see that